unit_test/test_spdm_requester/get_measurements.c

```diff
@@ -5415,12 +5402,13 @@ static void libspdm_test_requester_get_measurements_case35(void **state)
     assert_int_equal(status, LIBSPDM_STATUS_SUCCESS);
 
 #if LIBSPDM_RECORD_TRANSCRIPT_DATA_SUPPORT
-    assert_int_equal(spdm_context->transcript.message_m.buffer_size,
+    /* Subtract the size of the request message. */
```

Comment:
Why we need subtract the request message?

Reply:
I changed it so that the buffers contains everything (fill data, request, and response), and the expectation is that the sizes of the two buffers are the same as well as the contents.